src/collective/recaptcha/tests/test_settings.py

```diff
@@ -2,28 +2,27 @@
 from collective.recaptcha.settings import getRecaptchaSettings
 from collective.recaptcha.settings import IRecaptchaSettings
 from collective.recaptcha.settings import RecaptchaSettings
-from collective.recaptcha.testing import COLLECTIVE_RECAPTCHA
+from collective.recaptcha.testing import COLLECTIVE_RECAPTCHA_INTEGRATION_TESTING
 from plone.registry.interfaces import IRegistry
 from zope.component import getUtility
 
 import unittest
 
 
 class TestSettings(unittest.TestCase):
-    layer = COLLECTIVE_RECAPTCHA
+    layer = COLLECTIVE_RECAPTCHA_INTEGRATION_TESTING
 
     def setUp(self):
         self.portal = self.layer["portal"]
         self.registry = getUtility(IRegistry)
 
-    def tearDown(self):
-        try:
-            settings = self.registry.forInterface(IRecaptchaSettings)
-        except KeyError:
-            settings = IRecaptchaSettings(self.portal)
-        del settings
-
     def test_annotation(self):
```

Comment:
This will probably conflict with your other PR (#25).
I would focus on merging that one before working on this.
What do you think?

Reply:
Yes this will give conflict. Ultimately I want to remove this test. I wanted to separate the PRs by subject and keep the tests working. I will fix in the PR that is merged last.

Reply:
@ale-rt  after the merge of #25 I did a rebase here and this test was removed. Can you review please?